tools/tertiary-analysis/sceasy/sceasy_convert.xml

```diff
@@ -74,6 +78,14 @@
           <option value="scale.data">scale.data</option>
         </param>
       </when>
+      <when value="anndata2seurat">
```

Comment:
I think that you also need to add an option in the conditional.param block (lines 42 to 46) for `anndata2seurat`.